datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -325,6 +339,50 @@ private static void configureMlModelResolvers(final RuntimeWiring.Builder builde
             );
     }
 
+    /**
+     * Configures resolvers responsible for resolving the {@link com.linkedin.datahub.graphql.generated.DataFlow} type.
+     */
+    private static void configureDataFlowResolvers(final RuntimeWiring.Builder builder) {
+        builder
+            .type("Owner", typeWiring -> typeWiring
+                .dataFetcher("owner", new AuthenticatedResolver<>(
+                    new LoadableTypeResolver<>(
+                        CORP_USER_TYPE,
+                        (env) -> ((Owner) env.getSource()).getOwner().getUrn()))
+                )
+            );
+    }
+
+    /**
+     * Configures resolvers responsible for resolving the {@link com.linkedin.datahub.graphql.generated.DataJob} type.
+     */
+    private static void configureDataJobResolvers(final RuntimeWiring.Builder builder) {
+        builder
+            .type("DataFlow", typeWiring -> typeWiring
```

Comment:
This should be "DataJob" type. Basically you are saying "here's how to resolve specific fields of the DataJob" type... In this case, how to resolve the related DataFlow